.github/ISSUE_TEMPLATE/bug-report.yaml

```diff
@@ -1,50 +1,42 @@
-name: Bug Report
+name: Bug Report Form
 description: Report unexpected behavior during a meeting session here
 body:
   - type: markdown
     attributes:
       value: |
-        Thanks for taking the time to fill out this bug report!
+        Please go through the existing [issues](https://github.com/aws/amazon-chime-sdk-js/issues) or [FAQs](https://aws.github.io/amazon-chime-sdk-js/modules/faqs.html) to see if your issue has already been addressed before.
```

Comment:
Thought @vidya-mohan wants to convert this to checkbox?

Reply:
Thanks for the catch! I have converted them to checkboxes now.